spec/components/schemas/message/whatsapp.ts

```diff
@@ -19,6 +19,20 @@ const all: SchemaObject = {
         },
         minItems: 1,
       },
+      idRef: {
+        type: 'string',
+        description: `Id of the message sent or received in the conversation channel to reference as a message reply.
+        <br>*Only applicable to [WhatsApp](#tag/WhatsApp) channel.*`,
+        example: 'ZYYxZz1xzzz-xXzz-xzXz-Zy3x-ZYYxZz1xzzz',
```

Comment:
Não tem problema gerar um uuid qualquer como exemplo https://www.uuidgenerator.net/
```suggestion
        example: '7390113b-e120-41b5-8a07-c4567726abc2',
```